applicationinsights/channel/contracts/MessageData.py

```diff
@@ -9,6 +10,19 @@ class MessageData(object):
     
     DATA_TYPE_NAME = 'MessageData'
     
+    PYTHON_LOGGING_LEVELS = {
```

Comment:
Perhaps we can make a constants file in contracts or channel. The primary thing is to get it out of any of the actual type files in contracts. In node.js we'll soon have /contracts/constants.ts and then /contracts/generated/*.ts as we're switching that over to fully autogenerated schema-based types (like this MessageData one). Placing this in its own file, even if it stays in the same folder will be good enough while none of it is autogenerated yet here